frmts/netcdf/netcdfdataset.cpp

```diff
@@ -1960,19 +2029,18 @@ CPLErr netCDFRasterBand::CreateBandMetadata(const int *paDimIds,
     //  BandPos1 = (nBand - BandPos0*(3*4)) / (4)
     //  BandPos2 = (nBand - BandPos0*(3*4)) % (4)
 
-    GDALPamRasterBand::SetMetadataItem("NETCDF_VARNAME", szVarName);
     int Sum = 1;
     if (nd == 3)
     {
         Sum *= panBandZLev[0];
     }
 
     // Loop over non-spatial dimensions.
-    int result = 0;
     int Taken = 0;
 
     for (int i = 0; i < nd - 2; i++)
     {
+        int result;
```

Comment:
isn't `result` being used uninitialized if the following condition is false?

Reply:
no, it is also initialized in the else branch at line 2055